src/library_exceptions.js

```diff
@@ -391,7 +391,7 @@ var LibraryExceptions = {
       // just pass through the thrown ptr
       {{{ makeStructuralReturn(['catchInfo.ptr', 0]) }}};
     }
-    var typeArray = Array.prototype.slice.call(arguments);
+    var typeArray = Array.from(arguments);
```

Comment:
This code path happens on every exception thrown, so maybe it should use the larger/faster older form? Not as critical as in GL as in the other discussion, but still it might matter.